SimpleMVVM/ViewControllers/ViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  ViewController.swift
+//  SimpleMVVM
+//
+//  Created by Дмитрий Станкевич on 10.02.22.
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    //MARK: - Varibles
+    private var viewModel = ViewModel()
+    private let edgeInsets = UIEdgeInsets(all: 20)
+    private let imageSize = CGSize(width: 300, height: 300)
+    private let buttonAndFieldSize = CGSize(width: 120, height: 30)
+    
+
+    //MARK: - UI Varibles
+    private lazy var imageView: UIImageView = {
+       let image = UIImageView()
+        image.clipsToBounds = true
+        image.contentMode = .scaleAspectFill
+        image.translatesAutoresizingMaskIntoConstraints = false
+        image.backgroundColor = .red
+        image.image = UIImage(named: "404")
+        
+        return image
+    }()
+    
+    private lazy var infoLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.numberOfLines = 1
```

Comment:
U can freely remove this line. 1 is a default value 
https://developer.apple.com/documentation/uikit/uilabel/1620539-numberoflines